heroku/react-hk-components
# Upgrading @types dependencies

## Why the change?
[GUS Ticket](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001KpRlXYAV/view)
To get this application completely compliant with React 18, I updated the @types dependencies. I also removed @typesdependencies that we were not using or became irrelevant with other updates. 

## How to verify the changes?
+ visit PR app 
+ make sure it works like production 
+ the tests pass 

## Note 
I decided not to update @types/jest in this PR because these changes require updates to babel dependencies and some rewriting of code. I will address this in my next PR where I am going to update babel. I also did not update  @types/storybook__addon_actions and @types/storybook__react because those will be taken care of in a subsequent PR in which I upgrade @storybook dependencies.
